packages/create-principles-disciple/src/i18n.ts

```
@@ -0,0 +1,176 @@
+    feature_flags: 'Feature flags',
+    story_a_demo: 'Story A demo',
+    ready: 'Ready.',
+    diagnostics: 'Diagnostics',
+    install_partial_complete: 'Install partially complete — runtime + CLI verified, but review console is not yet deliverable',
+    not_ready_for_release: 'Not ready for seed-customer release — owner review console is a release-blocking gap.',
+    install_failed: 'Install failed',
+    next_action: 'Next action',
+    uninstall_warning_title: '⚠️  Important notice:',
+    uninstall_warning_msg: 'Uninstall will only remove plugin system files, your personal data will be kept.',
+    workspace_files_preserved: 'The following workspace files will be preserved:',
+    workspace_dir: 'Workspace directory',
```

ESLint

[error] Duplicate key 'workspace_dir'.

(no-dupe-keys)